README.md

```diff
@@ -113,7 +113,7 @@ Search for bugs:
 
 SWT used to track ongoing development and issues in Bugzilla .
 
-- <https://bugs.eclipse.org/bugs/buglist.cgi?product=Platform&component=SWT>
+- <https://bugs.eclipse.org/bugs/buglist.cgi?component=SWT&product=Platform>
```

Comment:
Both links lead to the same page, what does this change fix?

Reply:
Yes, I completely agree with you, but in my opinion the second link is more correct

Reply:
Either the link should be removed, or the title and rubric changed to mention that BugZilla is no longer used and this is the link to old (archived) bugs.